### ISSUE TITLE
Please tell me how to import it into Unreal Engine 5.

### ISSUE THREAD

**Issue by balzac0425** [FlailingFog/KK-Blender-Porter-Pack#566]
After importing a 3d character model into blender using KKBP, is it possible to import the character model into Unreal Engine 5?

**Comment by SirGhazian** [comment]
Yes, it can. Have you ever tried to export the character as FBX, OBJ, or DAE? Because those are common object file types that UE5 can import.

**Comment by esfe3323** [comment]
probably if you prepare the model for unity it also works on unreal.

**Comment by balzac0425** [comment]
It's done! Thank you very much.
I would like to ask one more question: I was able to export Koikatsu's animation with SB3Utility and import it into Blender, but is it possible to bring the animation to Unity or Unreal Engine?
